src/StoreKeeper/WooCommerce/B2C/Imports/ProductImport.php

```diff
@@ -1039,11 +1041,49 @@ function ($attribute) {
                     return !key_exists('attribute_option_id', $attribute);
                 }
             );
-            foreach ($nonAttributeOptions as $attribute) {
```

Comment:
why the `$nonAttributeOptions` are removed? the old loop for them should stay how it was